azavea/pfb-network-connectivity
# Small fixes for the pre-analysis deploy

## Overview

While working on validating PR #789 and PR #790, I decided to go ahead and fix #526 (add a favicon) and #758 (increase max zoom).  Also, CI failed because the [quantile](https://github.com/tvondra/quantile) Postgres extension we're using started failing to build (#792).  So this makes a small change to the analysis container's Dockerfile that gets it operational again.

Resolves #792, #526, and #758.

UPDATED: I hadn't seen issue #792 when I ran into it (email filters still need tweaking, apparently), but that's the same error that broke CI.  I updated the above to make reference to the issue.

### Demo

Hard to show the analysis not crashing (though see [the test branch build](http://urbanappsci.internal.azavea.com/job/azavea/job/pfb-network-connectivity/job/test%252Fpr-testing/)), but here's a screenshot of the favicon and higher zoomlevel:
![image](https://user-images.githubusercontent.com/6598836/74476280-0af31900-4e77-11ea-9b75-7a11d89a257e.png)

### Notes

It probably makes sense to upgrade the base image of the analysis at some point.  It's likely to not require many (or maybe any) code modifications, and it could conceivably improve performance.  At the least, it would hopefully reduce the future incidence of unexpected build failures.

## Testing Instructions

- If you already have a VM, run (in the VM) `scripts/update` then also run `docker-compose build --no-cache analysis`.  If you don't have a VM, just running `scripts/setup` will make one and there will be no cache to skip.
- Run `scripts/server` (in the VM) to see the front-end changes.

## Checklist
